packages/deployment/upgrade-test/Makefile

```diff
@@ -1,6 +1,7 @@
 REPOSITORY = agoric/upgrade-test
 # use :dev (latest prerelease image) unless we build local sdk
 DEST_IMAGE ?= $(if $(findstring local_sdk,$(MAKECMDGOALS)),ghcr.io/agoric/agoric-sdk:latest,ghcr.io/agoric/agoric-sdk:dev)
+DEST_PLAN ?=
```

Comment:
does `?=` do anything when there's nothing following it?  As I read the manual, `?=` sets the variable to the following value if it's not already set.

Reply:
That was a (now removed) stray line, and you're right, it didn't have any effect anyway.